docs/api.json

```diff
@@ -367,7 +367,7 @@
             {
                 "is_required": false, 
                 "type": "int", 
-                "name": "amount<amount>", 
+                "name": "amount", 
```

Comment:
The angle brackets need to be removed so my API Code generation can work properly